modules/virt-hot-plugging-cpu.adoc

```diff
@@ -17,5 +17,10 @@ You can increase or decrease the number of CPU sockets allocated to a virtual ma
 . Select the *vCPU* radio button.
 . Enter the desired number of vCPU sockets and click *Save*.
 +
+[NOTE]
+====
+You can hot-plug up to three times the default initial number of vCPU sockets of the VM. Exceeding this limit requires a restart.
```

Comment:
[The IBM Style Guide](https://www.ibm.com/docs/en/ibm-style?topic=measurement-numerals-versus-words) also recommends generally using numerals versus words for numbers - suggestion to take or leave